docs/get_involved.rst

```diff
@@ -0,0 +1,9 @@
+Getting Involved
```

Comment:
you probably want to adopt a policy around title capitalization - in Envoy we are now moving towards "normal capitalization" - ie more like what you have below in:

```rst

Welcome to the Envoy Gateway documentation!
===========================================
```

ie - only the initial word and proper nouns are capitalized

doing what you have here "Where Every Word Is Capitalized" loses emphasis for proper nouns and throws up many more questions than it resolves - so my suggestion is to adopt normal caps

Reply:
My own preference is actually title case, but I think conforming to the Envoy style makes sense here.